src/global-styles.tsx

```diff
@@ -0,0 +1,130 @@
+import { darkTheme, lightTheme, useThemeStore } from 'features/theming';
+import { css, Global } from '@emotion/react';
+
+const globals = css`
+  :root {
+    font-family: Inter, Avenir, Helvetica, Arial, sans-serif;
+    font-size: 16px;
+    line-height: 24px;
+    font-weight: 400;
+
+    color-scheme: light dark;
+    font-synthesis: none;
+    text-rendering: optimizeLegibility;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    -webkit-text-size-adjust: 100%;
+  }
+
+  html,
+  body,
+  #root {
+    height: 100%;
+  }
+
+  #root {
+    overflow: auto;
+  }
+
+  body {
+    background-color: var(--bg);
+  }
+`;
+
+// ref: https://www.joshwcomeau.com/css/custom-css-reset/#the-css-reset
+const reset = css`
+  /**
+   * 1. Use a more-intuitive box-sizing model.
+   */
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+  }
+
+  /**
+   * 2. Remove default margin
+   */
+  * {
+    margin: 0;
+  }
+
+  /**
+   * 3. Allow percentage-based heights in the application
+   */
+  html,
+  body {
+    height: 100%;
+  }
+
+  /**
+   * Typographic tweaks!
+   * 4. Add accessible line-height
+   * 5. Improve text rendering
+   */
+  body {
+    line-height: 1.5;
+    -webkit-font-smoothing: antialiased;
+  }
+
+  /**
+   * 6. Improve media defaults
+   */
+  img,
+  picture,
+  video,
+  canvas,
+  svg {
+    display: block;
+    max-width: 100%;
+  }
+
+  /**
+   * 7. Remove built-in form typography styles
+   */
+  input,
+  button,
+  textarea,
+  select {
+    font: inherit;
+  }
+
+  /**
+   * 8. Avoid text overflows
+   */
+  p,
+  h1,
+  h2,
+  h3,
+  h4,
+  h5,
+  h6 {
+    overflow-wrap: break-word;
+  }
+
+  /**
+   * 9. Create a root stacking context
+   */
+  #root,
+  #__next {
+    isolation: isolate;
+  }
+`;
```

Comment:
I'm not dead set on this. What ended up being nice here was the ability to import this component in a decorator in storybook and toggle the body background based on the theme.

Reply:
Yeah, this is a bit odd, but it seems reasonable. Emotion is handling your CSS in JS, why not let it handle the rest?